e2e/cypress/support/renkulab-fixtures/projects.ts

```diff
@@ -41,44 +41,55 @@ function Projects<T extends FixturesConstructor>(Parent: T) {
       return this;
     }
 
-    interceptMigrationCheck(name, fixture) {
+    project(useFixture = true, path = "", name = "getProject", result = "projects/project.json") {
+      const fixture = useFixture ? { fixture: result } : undefined;
+      cy.intercept(
+        `/ui-server/api/projects/${encodeURIComponent(path)}?statistics=true`,
+        fixture
+      ).as(name);
+      return this;
+    }
+
+    interceptMigrationCheck(name, fixture, params) {
```

Comment:
I suggest changing the signature of interceptMigrationCheck to 
```suggestion
    interceptMigrationCheck(name, fixture, params=null) {
```
Then the params could be defaulted if not provided. Otherwise it has to be passed in through a long call-chain.